packages/deploy/src/core/CoreContracts.ts

```diff
@@ -771,163 +769,189 @@ export default class EvmCoreDeploy extends AbstractCoreDeploy<config.EvmCoreCont
 
     //  ========= Home =========
     // Home upgrade setup contracts are defined
-    checklist.assertBeaconProxy(this.data.home, 'Home');
+    checklist.assertBeaconProxy(this.data.home, 'Home beacon proxy is defined');
 
     // updaterManager is set on Home
     const updaterManager = await this.home.updaterManager();
     checklist.equalIds(
       updaterManager,
       this.data.updaterManager,
-      'UpdaterManager',
+      'UpdaterManager is configured on Home',
     );
 
     // state
     const state = await this.home.state();
-    checklist.equals(1, state, `Home state`);
+    checklist.equals(1, state, `Home state is Active`);
```

Comment:
```suggestion
    checklist.equals(1, state, `Home state is ${state}`);
```

I like this as an implicit test

Reply:
"Home state is ${state}, expected Active"